src/components/apps/queries.js

```diff
@@ -51,22 +47,20 @@ export function useReferenceGenomes(enabled, onSuccess, onError) {
     return useQuery({
         queryKey: REFERENCE_GENOMES_QUERY_KEY,
         queryFn: getReferenceGenomes,
```

Comment:
Not sure how this changed, but in prod this calls the API with `/api/reference-genomes?deleted=false`, but in local testing, this branch is now making the API call `/api/reference-genomes?deleted=undefined`, which returns an error from the service: `{ deleted: "(not (instance? java.lang.Boolean \"undefined\"))" }`

So I guess this needs to explicitly set the `deleted` param now?
```suggestion
        queryFn: () => getReferenceGenomes({ deleted: false }),
```